integration/src/test/java/edu/unc/lib/boxc/integration/factories/FileFactory.java

```diff
@@ -34,30 +34,32 @@
 import java.util.Objects;
 
 /**
+ * Factory for creating test FileObjects
  * @author snluong
  */
 public class FileFactory extends ContentObjectFactory {
-    public static final String FILE_FORMAT = "fileFormat";
-    public static final String IMAGE_FORMAT = "image";
-    public static final String TEXT_FORMAT = "text";
-    public static final String PDF_FORMAT = "pdf";
-    public static final String AUDIO_FORMAT = "audio";
-    public static final String VIDEO_FORMAT = "video";
+    public static final String FILE_FORMAT_OPTION = "fileFormat";
```

Comment:
Sorry, I was thinking it would be `FILE_FORMAT_OPTION` and the others would stay `IMAGE_FORMAT`, etc, so that it was more clear that FILE_FORMAT_OPTION was a key in the options object, while IMAGE_FORMAT was a value.